### ISSUE TITLE
How do we want to handle failed remediation jobs?

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/scholarsphere#1907]
There will inevitably be failed remediation jobs.  They could happen for a number of reasons: page limit quotas, PDF API failures, PDF Accessibility application errors, and so on.  The API _should_ report back the error (assuming the error doesn't break the notification system).  Depending on the error, we could handle these in different ways.  For instance, if page quotas are exceeded, we would likely just need to increase our limit, so we may just want to automatically reset timestamps and flags to allow pdf remediation to be triggered for this file again.

**Comment by ajkiessl** [comment]
Related #1935
